pkg/deploy/generator/resources.go

```diff
@@ -556,6 +558,66 @@ func (g *generator) lb() *arm.Resource {
 	}
 }
 
+func (g *generator) lbAlerts() []*arm.Resource {
+	return []*arm.Resource{
+		{
+			Resource: mgmtmonitor.ActionGroupResource{
```

Comment:
Each one of these has to be onboarded separately?  Can we do one per subscription?  Can we even do one across multiple subscriptions?  I've given you temporary access to another Azure subscription ("Azure Red Hat OpenShift INT (RH Engineering)") - can you test to see if MetricAlertResources can references ActionGroupResources in different subscriptions?

Reply:
Each Action Group has to be connected separately.
We can reuse the Action Group for multiple alerts, the AlertRule (the name of the alert) and AlertTargetId (resource id) variables are passed on, but I'm not sure how it's presented in a ticket (more variables are in a table in the docs https://icmdocs.azurewebsites.net/developers/Connectors/IcMAction.html).

Found information for cross-subscription, and checked that it errors when attempting to create a cross-sub metrics alert->actiongroup connection:
"For metric and log alerts, action groups selected must be in the alert rule’s subscription. For activity log alerts, action groups can be selected from subscriptions other than the alert rule’s subscription."
So in our case neither logs nor metrics based alerts will work unless we want to track resource creation/deletion.

Reply:
ok, in this case the action group should be put into yet another new ARM deployment template.  It'll be like rp-global-production.json, except that whereas that is shared by all RPs globally, your new one will be deployed in each RP subscription.

copy all the deploy code associated with rp-global-production.json, so that you do the deployment to the subscription of the RP, but to a shared resource group.  That means one AG onboard per subscription, instead of per RP.

^ probably doesn't make too much sense, ping @mjudeikis with any questions ;-)

Reply:
Moved the action group to a separate deploy/rp-per-subscription-production.json.

Side note: The 'rp-production-global.json' is not mentioned in documentation. Should it go in right before the rp-production.json?